src/main/java/core/mate/academy/service/producerimpl/ExcavatorProducerImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service.producerimpl;
+
+import core.mate.academy.model.Excavator;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class ExcavatorProducerImpl implements MachineProducer<Excavator> {
+    @Override
+    public List<Excavator> get() {
+        List<Excavator> excavators = new ArrayList<>();
+        IntStream.range(1, 10)
+                .forEach(i -> excavators.add(new Excavator()));
```

Comment:
you added a custom property to `Excavator` but you don't initialize it here - consider declaring another constructor